### ISSUE TITLE
changelog fails on version lines

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2054]
Web-UI presentation of pertinent rockstor changelog entries fails on version lines:

E.g.:
Given the following version line within an available rockstor update rpm package:
```
* Sun Feb 24 2019 Philip Guyton <philip@yewtreeapps.com> - 3.9.2-48
```
The resulting 'Software Updates' page counterpart displays:
```
# @
```
This is due to incomplete parsing such that package version lines are treated as change item entries for historical reasons.

See template file:
https://github.com/rockstor/rockstor-core/blob/master/src/rockstor/storageadmin/static/storageadmin/js/templates/update/version_info.jst
and associated view js file:
https://github.com/rockstor/rockstor-core/blob/master/src/rockstor/storageadmin/static/storageadmin/js/views/version.js

i.e. in latter we have:
            changeMap: this.changeLog(this.changeList),

Hence changelog in latter needs some attention for this issues resolution.

**Comment by phillxnet** [comment]
Tracking this further it seems we also don't pass the problematic lines to the above referenced view related files as logging indicates that the "# @" Web-UI elements come form blank entries returned from:

/api/commands/update-check

https://github.com/rockstor/rockstor-core/blob/master/src/rockstor/storageadmin/static/storageadmin/js/views/version.js#L67

**Comment by phillxnet** [comment]
I have a local working solution to this issue. Noting to avoid duplication of effort.
Pull request to follow.

**Comment by phillxnet** [comment]
Example of pending pr output given pertinent changelog content within proposed package:
(Actual changelog used is a work in progress as further merges are planned soon)
```
List of changes in this update
    * Tue Jul 9 13:00:00 2019 Philip Guyton - 3.9.2-49
    -Fix ENV display in RockonInstall_summary-table. Fixes #2029 @FroggyFlox
    -Fix size sorting on snapshots / shares pages. Fixes #1368 #1878 @Psykar
    -honour tag element within rock-on json. Fixes #2014 @phillxnet
    -pool resize disk removal unknown internal error and no UI counterpart. Fixes #1722 @phillxnet
    -[openSUSE] don't rely on PGDATA env var. Fixes #2041 @phillxnet
    * Sun Feb 24 12:00:00 2019 Philip Guyton - 3.9.2-48
    -Skip empty label fields. Fixes #2018 @FroggyFlox
    -Implement Add Labels feature for already-installed Rock-Ons. Fixes #1998 @FroggyFlox
    -regression in unit tests - environment outdated since 3.9.2-46. Fixes #1993 @phillxnet
    * Sun Feb 3 12:00:00 2019 Suman Chakravartula - 3.9.2-47
...
```
And via screen grab:

![Changelog-Web-UI-fix](https://user-images.githubusercontent.com/2521585/60994675-dfd8a200-a348-11e9-9e5b-90a7b71609ba.png)
